docs/source/error_code_list2.rst

```diff
@@ -258,6 +258,19 @@ except that attempting to invoke an undefined method (e.g. ``__len__``) results
 while attempting to evaluate an object in boolean context without a concrete implementation results in a truthy value.
 
 
+Check that function isn't used in boolean context [truthy-function]
```

Comment:
Should be under “enabled by default” rather than “optional checks”, right?

- #14380